CMakeLists.txt

```diff
@@ -115,8 +115,8 @@ else()
     enable_language(ASM)
     try_compile(PQ_ASM_COMPILES ${CMAKE_BINARY_DIR}
             SOURCES
-            "${CMAKE_SOURCE_DIR}/tests/unit/s2n_pq_asm_noop_test.c"
-            "${CMAKE_SOURCE_DIR}/pq-crypto/sike_r2/fp_x64_asm.S")
+            "${CMAKE_CURRENT_LIST_DIR}/tests/unit/s2n_pq_asm_noop_test.c"
```

Comment:
this shouldn't have any effect. Could you try running without the cmake var? it already defaults to the same directory as the current CMakeLists.txt file.

Reply:
Tried removing the CMake Environment variable and the build started failing. 
```
+ cmake -DCMAKE_CXX_COMPILER=g++-9 -DCMAKE_C_COMPILER=gcc-9 -DCMAKE_PREFIX_PATH=/home/ANT.AMAZON.COM/aweibel/workspace/aws-sdks/local-c-install -DCMAKE_INSTALL_PREFIX=/home/ANT.AMAZON.COM/aweibel/workspace/aws-sdks/local-c-install -DBUILD_SHARED_LIBS=ON -DBUILD_DEPS=OFF -DCMAKE_BUILD_TYPE=Debug ../
-- The C compiler identification is GNU 9.2.1
-- Check for working C compiler: /usr/bin/gcc-9
-- Check for working C compiler: /usr/bin/gcc-9 -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- Attempting to try_compile PQ ASM
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/gcc-9
CMake Error at /home/ANT.AMAZON.COM/aweibel/workspace/github/s2n/build/CMakeFiles/CMakeTmp/CMakeLists.txt:15 (add_executable):
  Cannot find source file:

    tests/unit/s2n_pq_asm_noop_test.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx
CMake Error at /home/ANT.AMAZON.COM/aweibel/workspace/github/s2n/build/CMakeFiles/CMakeTmp/CMakeLists.txt:15 (add_executable):
  No SOURCES given to target: cmTC_69e57
CMake Error at CMakeLists.txt:116 (try_compile):
  Failed to generate test project build system.
```